cmd/cmdtest/testutils.go

```diff
@@ -209,3 +209,67 @@ func AddLocalFileRepo(repoName string, repoFilePath string) (string, func(), err
 
 	return repoURL, cleanupFunc, err
 }
+
+// RunDockerCmdExec runs the docker command with the given args in a new process
+// The stdout and stderr are captured, printed, and returned
+// args will be passed to the docker command
+// workingDir will be the directory the command runs in
+func RunDockerCmdExec(args []string, workingDir string) (string, error) {
+	execDir, err := os.Getwd()
```

Comment:
`docker` commands don't rely on the working directory, so you can remove all the code related to this.